scripts/in_container/airflow_ci.cfg

```diff
@@ -44,12 +44,6 @@ smtp_mail_from = airflow@example.com
 broker_url = amqp://guest:guest@rabbitmq:5672/
 result_backend = # overridden by startup scripts
 
-[celery_broker_transport_options]
```

Comment:
This is moved out of here in to default_test.cfg so that the config tests pass outside of breeze.

Reply:
Actually, scratch this. I'm going to  revert this change.